build_tools/bazel/build_core.sh

```diff
@@ -42,6 +42,7 @@ fi
 declare -a test_env_args=(
   --test_env=IREE_LLVMJIT_DISABLE=$IREE_LLVMJIT_DISABLE
   --test_env=IREE_VULKAN_DISABLE=$IREE_VULKAN_DISABLE
+  --action_env=IREE_LLVMAOT_LINKER_PATH=$IREE_LLVMAOT_LINKER_PATH
```

Comment:
Please add this to the env for all the bazel builds, most importantly the bindings build, which also uses RBE. The environment is part of the cache key, which means we want this to be the same everywhere to get cache hits.